### ISSUE TITLE
[JOSS REVIEW] linux-gnu build info is distro specific

### ISSUE THREAD

**Issue by zingale** [enzo-project/enzo-dev#96]
Upon running 
```
make machine-linux-gnu
```
it outputs:
```
 *** Execute 'gmake clean' before rebuilding executables ***

   MACHINE: Use apt-get to install libhdf5-serial-dev gfortran openmpi-bin libopenmpi-dev
   MACHINE-NAME: linux-gnu
```

but many Linux distros do not use `apt-get` and new users could be confused.  It would be helpful to have a list of the requirements needed for Enzo in the README.md or a REQUIREMENTS.txt

**Comment by zingale** [comment]
JOSS Review: https://github.com/openjournals/joss-reviews/issues/1636

**Comment by bwoshea** [comment]
@zingale  This is addressed in PR #115

**Comment by zingale** [comment]
looks good!
